byron-proxy/byron-proxy.cabal

```diff
@@ -108,3 +114,43 @@ executable byron-proxy
   ghc-options:         -threaded
                        -fwarn-incomplete-patterns
                        -fwarn-unused-imports
+
+executable validator
+  main-is:             Validator.hs
+  other-modules:       Logging
+  -- other-extensions:
+  build-depends:       base,
```

Comment:
I copied this from `executable byron-proxy` above. It's possible that many of these are not actually needed. Is there a tool that can identify them?

Reply:
`weeder` should be able to help here